subprojects/packagefiles/fmt/meson.build

```diff
@@ -27,6 +27,7 @@ fmt_dep = declare_dependency(
     compile_args : fmt_interface_cpp_args,
     link_with : fmt_lib
 )
+meson.override_dependency('fmt', fmt_dep)
```

Comment:
This should be guarded with `if meson.version().version_compare('>=0.54.0')` to make sure it's only evaluated on versions of Meson that are new enough to have this method.

Reply:
Fixed.